lib/plugins/create/templates/openwhisk-nodejs/README.md

```diff
@@ -0,0 +1,58 @@
+# Serverless OpenWhisk Node.js Template
+
+Hello! 😎
+
+This is a template Node.js service for the OpenWhisk platform. Before you can deploy your service, please follow the instructions below…
+
+### Have you set up your account credentials?
+
+Before you can deploy your service to OpenWhisk, you need to have an account registered with the platform.
+
+- *Want to run the platform locally?* Please read the project's [*Quick Start*](https://github.com/openwhisk/openwhisk#quick-start) guide for deploying it locally.
+- *Want to use a hosted provider?* Please sign up for an account with [IBM Bluemix](https://console.ng.bluemix.net/) and then follow the instructions for getting access to [OpenWhisk on Bluemix](https://console.ng.bluemix.net/openwhisk/). 
+
+Account credentials for OpenWhisk can be provided through a configuration file or environment variables. This plugin requires the API endpoint, namespace and authentication credentials.
+
+**Do you want to use a configuration file for storing these values?** Please [follow the instructions](https://console.ng.bluemix.net/openwhisk/cli) for setting up the OpenWhisk command-line utility. This tool stores account credentials in the `.wskprops` file in the user's home directory. The plugin automatically extracts credentials from this file at runtime.  No further configuration is needed.
+
+**Do you want to use environment variables for credentials?** Use the following environment variables to be pass in account credentials. These values override anything extracted from the configuration file.
+
+- *OW_APIHOST* - Platform endpoint, e.g. `openwhisk.ng.bluemix.net`
+- *OW_AUTH* - Authentication key, e.g. `xxxxxx:yyyyy
+
+
+
+### Have you installed and setup the provider plugin?
+
+Using the framework with the OpenWhisk platform needs you to install the provider plugin and link this to your service. 
+
+####  Install the provider plugin
+
+```
+$ sudo npm install --global serverless-openwhisk
+```
+
+*Due to an [outstanding issue](https://github.com/serverless/serverless/issues/2895) with provider plugins, the [OpenWhisk provider](https://github.com/serverless/serverless-openwhisk) must be installed as a global module.*
+
+
+#### Link provider plugin to service directory
+
+Using `npm link` will import the provider plugin into the service directory. Running `npm install` will automatically perform this using a `post install` script.
+
+```
+$ npm link serverless-openwhisk
+or
+$ npm install
+```
+
+
+
+**_…and that's it!_**
+
+### Deploy Service
+
+Use the `serverless` command to deploy your service. The sample `handler.js` file can be deployed without modification.
+
+```shell
+serverless deploy
+```
```

Comment:
Just an idea. Should we add a quick info about the Plugin repo here so that users know where they can find more information / file issues?